ocs_ci/run_ocsci.py

```diff
@@ -1,11 +1,11 @@
 import sys
 
-import ocsci.main
+from .framework import main as runner
```

Comment:
I think it would be better to keep absolute importing everywhere

Reply:
Done